### DISCUSSION TITLE
Is the docs about contribution up to date ?

### DISCUSSION THREAD

**Discussion by joaopfg** [encode/starlette#1847] [Q&A] [answered]
I'm following the instructions [here](https://www.starlette.io/contributing/) and getting problems with the line `scripts/install`. Is it up to date ?

The error I get is:

```
===============================
Unsupported installation method
===============================
Starlette no longer supports installation with `python setup.py install`.
Please use `python -m pip install .` instead.
```

**Comment by Kludex** [comment]
I'm not able to run `scripts/install` locally, but with another error (wrong pip and setuptools version). So... I guess the answer is: no.

PR welcome to fix it.

**Reply by Kludex** [reply]
I can't reproduce what I said here.

**Comment by joaopfg** [comment]
@Kludex Ok, I will propose a PR soon

**Reply by Kludex** [reply]
Same PR welcome in `httpx`, and `uvicorn`. JFYK

**Comment by joaopfg** [comment]
PR is [open](https://github.com/encode/starlette/pull/1848)

**Comment by Kludex** [comment]
I'm not able to reproduce your issue. :disappointed:

**Comment by Kludex** [comment, marked as answer]
This is fixed on master.
